go.mod

```diff
@@ -96,3 +98,5 @@ require (
 // that we do not rely upon. It appears to introduce performance regressions
 // for us.
 exclude github.com/go-sql-driver/mysql v1.6.0
+
```

Comment:
Also, why is there an empty line between these excludes?

Also, since https://github.com/letsencrypt/boulder/pull/6976 just landed, this exclude should be obsolete.